src/conventions/gmsh.jl

```diff
@@ -0,0 +1,102 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+"""
+Ordering convention of the GMSH project:
+https://gmsh.info/doc/texinfo/gmsh.html#Node-ordering
+"""
+struct GMSH <: OrderingConvention end
+
+# Triangle
+# v
+# ^                 
+# |                 
+# 2                 
+# |`\               
+# |  `\             
+# |    `\           
+# |      `\         
+# |        `\       
+# 0----------1 --> u
+
+connectivity(::Type{<:Triangle{GMSH}}, ::Val{1}, ::Type{GMSH}) = connect.([(1, 2), (2, 3), (3, 1)], Segment)
+
+# Quadrangle
+#       v
+#       ^
+#       |
+# 3-----------2      
+# |     |     |      
+# |     |     |      
+# |     +---- | --> u
+# |           |      
+# |           |      
+# 0-----------1      
+
+connectivity(::Type{<:Quadrangle{GMSH}}, ::Val{1}, ::Type{GMSH}) = connect.([(1, 2), (2, 3), (3, 4), (4, 1)], Segment)
```

Comment:
Inconsistent formatting. Some functions you break the line after the equal sign, and some others like this you produce a really long line. Can you please add a breakline after the equal sign?